### PR TITLE
Make the use of Geoserver optional

### PR DESCRIPTION
**Pull request by ManuelStardust**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ El nombre sugerido para la carpeta a asociar a la variable GS_DATA_DIR es 'geose
    - MODEL_OUPUTS_DIR: ruta a la carpeta en el host donde se almacenaran los resultados de la ejecución del simulador (puede ser relativa a la ubicación del archivo docker-compose.yml de este repositorio)
    - MODEL_PASSWORD: contraseña para el usuario que ejecuta el simulador
 
-1. Puede correr el sistema de 2 formas: con postgres o con oracle.
+2. Puede correr el sistema de 2 formas: con postgres o con oracle.
```
Comment (erikasv) [inline README.md:32]: Using always 1 makes it easier to introduce new steps without having to change the numeration, in the visualization is changed by the parser automatically (but you don't need to change it back, I only tell you for you to know)

**Comment by erikasv** [comment]
Please connect the issue

File: docker-compose.override.geoserver.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    environment:
+      - GEOSERVER_ADMIN_USER=${GS_USER}
+      - GEOSERVER_ADMIN_PASSWORD=${GS_PASS}
+    networks:
+      - server_net
+
+
+  tool_server:
+    environment:
+      - GS_ENABLE=true
```
Comment (erikasv) [inline docker-compose.override.geoserver.yml:25]: Please add the newline at the end

File: src/app.js [hunk 3]
```diff
@@ -87,11 +87,15 @@ app.post(
...
       const workspaceName = `${workspace}_${body.module}`;
 
-      // Load to Geoserver
-      const zip_path = await compress_file(workspaceName, shp_folder);
-      await create_workspace(workspaceName);
-      await create_datastore(workspaceName, shp_name, zip_path);
+      const { GS_ENABLE } = process.env;
 
+      // Load to Geoserver
+      if(GS_ENABLE==="true"){
```
Comment (erikasv) [inline src/app.js:93]: Please run prettier

**Review by erikasv — COMMENTED** [review]
Please add in the readme that the GS_ environment variables are only needed if you're running GeoServer

File: src/app.js (hunk 3 above)
Comment (ManuelStardust) [reply src/app.js]: Done

File: README.md (hunk 1 above)
Comment (ManuelStardust) [reply README.md:34]: Ok, thanks

File: docker-compose.override.geoserver.yml (hunk 2 above)
Comment (ManuelStardust) [reply docker-compose.override.geoserver.yml]: Done

**Comment by ManuelStardust** [comment]
> Please add in the readme that the GS_ environment variables are only needed if you're running GeoServer

Done

**Review by erikasv — APPROVED** [review]

**Pull request merged**